spec/features/session.rb

```diff
@@ -0,0 +1,20 @@
+module SessionHelpers
```

Comment:
seems like this file is not being used - to use it you would need to 'include' it in the correct places in your spec files